src/player/Player.js

```diff
@@ -49,32 +35,47 @@ class Player {
   }
 
   render() {
-    updateSpeedometer(this.vx, this.ax)
+    updateSpeedometer(this.vx, this.ax, this.vy)
     this.vx += this.ax
     this.x += this.vx
+    this.y += this.vy
     if (rocket.velocity < 7) {
       rocket.x = this.x + rocketDefaultX
     }
 
+    if (rocket.velocity < 7) {
+      rocket.y = this.y + rocketDefaultY
+    }
+
     if (this.x <= 0) {
       this.x = 0
       this.vx = 0
       this.ax = 0
     }
 
+    if (this.y <= 0) {
+      this.y = 0
+    }
+
     if (this.x + this.width >= screen.width) {
       this.x = screen.width - this.width
       this.vx = 0
       this.ax = 0
     }
 
+    if (this.y + this.height >= screen.height) {
```

Comment:
If player is below the screen, game is over.
Do we need to reset anything?

I think it will work without this `if ()`.